### PR TITLE
Introduce Cody as IntelliJ icon

### PR DESCRIPTION
**Pull request by rickie**
As requested by @timtebeek 😉.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,6 @@ target
 
 # The Git repositories checked out by the integration test framework.
 integration-tests/.repos
+
+# Specific file exclusions.
```
Comment (Stephan202) [inline .gitignore:16]: So it's an _inclusion_ in the repo ;). Perhaps we can just move this to line 7, so that a comment isn't necessary?

**Review by Stephan202 — COMMENTED** [review]
Nice idea. I can't find the menu entry to change this feature (perhaps because I didn't switch to the new UI yet), but I wonder whether we:
1. Could/should use a Cody image (perhaps still with a Picnic reference).
2. Use an SVG file ([seems so](https://youtrack.jetbrains.com/issue/IDEA-325249)).

**Review by Stephan202 — COMMENTED** [review]
Rebased and dropped the `\r` characters from the SVG file. I tried out the new IDEA UI, and indeed saw the logo :+1:

File: .gitignore [hunk 2]
```diff
@@ -3,7 +3,8 @@
 .classpath
 .DS_Store
 .factorypath
-.idea
+**/.idea/*
```
Comment (Stephan202) [inline .gitignore:6]: How does this apply to this PR / under which circumstances may this happen? :)

Would expect that this suffices: 
```suggestion
**/.idea
```

Comment (rickie) [reply .gitignore:6]: I did this as I got some changes in the `integration-test` directory of Checkstyle and the `.idea` folder. I made this change and it went away.

Comment (rickie) [reply .gitignore:6]: The `/*` part was required as we have the inclusion afterwards.

**Review by Stephan202 — APPROVED** [review]
Nice! GitHub claims that Cody has a bunch of empty space at the top and bottom, but IDEA and my local image viewer claim that this isn't so :+1:.

Don't want to block the PR over a `.gitattributes` discussion, but do want to understand :grimacing:

**Review by mohamedsamehsalah — APPROVED** [review]

**Comment by mohamedsamehsalah** [comment]
Super cool 🤩 
I would personally love to see Picnic icon added as sugegsted by Stephan ☝️ 

Screenshot 2024-03-25 at 15 45 40

**Comment by rickie** [comment]
> I would personally love to see a Picnic reference on the icon added as sugegsted by Stephan ☝️

Yeah we of course have the :picnic-cody: but I think it's not that visible if we would add that. So I would vote to leave it as-is.

**Comment by rickie** [comment]
Circled back a bit late; but verified that with the recent changes it works :).

**Comment by rickie** [comment]
Suggested commit message:
```
Introduce Picnic logo as IntelliJ icon (#1075)
```

**Comment by rickie** [comment]
@Stephan202 feel free to merge this one :).

**Pull request merged**